pkg/scheduler/actions/allocate/allocate.go

```diff
@@ -208,6 +207,11 @@ func (alloc *Action) allocateResources(queues *util.PriorityQueue, jobsMap map[a
 			stmt.Commit()
 		}
 
+		// For FIFO, if a job fails to allocate (stmt == nil), block the queue by not re-queuing it.
+		// This prevents subsequent jobs in the same queue from being scheduled in the current cycle.
+		if queue.DequeueStrategy == api.DequeueStrategyFIFO && stmt == nil {
```

Comment:
Seems if a job failed to allocate, the whole queue will not be re-enqueued, this is not reasonable.

Reply:
Thanks for your comments， I'll fix this.